packages/react/src/components/Tooltip/DefinitionTooltip.jsx

```diff
@@ -0,0 +1,228 @@
+import cx from 'classnames';
```

Comment:
If this was copied from Carbon and modified, please indicate so in here as a comment